webapp/web/.gcloudignore

```diff
@@ -1,28 +1,35 @@
-# Don't upload test data, package config, etc. to AppEngine.
-**/*_test.go
+# Tests
+*_test.go
+test/
+webdriver/
+static/24278ab617/
+static/wptd-metrics/
+
+# Configurations
 .eslintrc.json
+package-lock.json
+package.json
+renovate.json
+wct.conf.json
+
+# Misc
 .git/
 .github/
+.lighthouseci/
 .vscode/
-build/
 CODE_OF_CONDUCT.md
-components/test/
-docs/
-git/
-ISSUE_TEMPLATE/
+CONTRIBUTING.md
 ISSUE_TEMPLATE.md
-LICENSE
-package-lock.json
-package.json
+ISSUE_TEMPLATE/
 PULL_REQUEST_TEMPLATE.md
 README.md
+docs/
+git/
+
+# Other services
+api/query/cache/service/
 results-processor/
-revisions/
-static/24278ab617/
-static/wptd-metrics/
-test/
-wct.conf.json
-webdriver/
+util/
```

Comment:
util/ will ignore https://github.com/web-platform-tests/wpt.fyi/blob/master/webapp/components/test/util/helpers.js too?

Reply:
actually that is a part of test/.